sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/StringExpressionsSuite.scala

```diff
@@ -725,4 +725,43 @@ class StringExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
     checkEvaluation(FindInSet(Literal("abf"), Literal("abc,b,ab,c,def")), 0)
     checkEvaluation(FindInSet(Literal("ab,"), Literal("abc,b,ab,c,def")), 0)
   }
+
+  test("ParseUrl") {
+    def checkParseUrl(expected: String, urlStr: String, partToExtract: String): Unit = {
+      checkEvaluation(
+        ParseUrl(Literal.create(urlStr, StringType), Literal.create(partToExtract, StringType)),
+        expected)
+    }
+    def checkParseUrlWithKey(expected: String, urlStr: String,
+                      partToExtract: String, key: String): Unit = {
+      checkEvaluation(
+        ParseUrl(Literal.create(urlStr, StringType), Literal.create(partToExtract, StringType),
+                 Literal.create(key, StringType)), expected)
```

Comment:
For this one, 2-indentation.

https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide